yb-voyager/cmd/importSchema.go

```diff
@@ -109,7 +110,25 @@ func importSchema() {
 		}
 	}
 
+	importDefferedStatements()
 	log.Info("Schema import is complete.")
+
+	if len(failedSqlStmts) > 0 {
+		failedSqlFilePath := filepath.Join(exportDir, "schema", "failed.sql")
+		file, err := os.OpenFile(failedSqlFilePath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+		if err != nil {
+			utils.ErrExit("open file: %v", err)
+		}
+
+		for i := 0; i < len(failedSqlStmts); i++ {
+			_, err = file.WriteString(failedSqlStmts[i].formattedStmt + "\n\n")
+			if err != nil {
+				utils.ErrExit("failed writing in file %s: %v", failedSqlFilePath, err)
+			}
+		}
```

Comment:
Define and use a separate function `dumpStatements(failedSqlStmts, "failed.sql")`.

Reply:
done